prow/cmd/deck/static/script.js

```diff
@@ -528,6 +528,19 @@ function redraw(fz) {
             r.appendChild(createTextCell(""));
             r.appendChild(createTextCell(""));
         }
+        if (spyglass) {
+            if (build.url.startsWith("https://k8s-gubernator.appspot.com/build/")) {
```

Comment:
OMG ... of course this doesn't work! This is only your Gubernator. Please fix to either always replace or allow for configuring the Gubernator URL

Reply:
🤦‍♂️ /shrug

Reply:
@stevekuznetsov I opened issue #9403 for this